Source/Fuse.Scripting.JavaScript/RootableScriptModule.uno

```diff
@@ -27,8 +27,11 @@ namespace Fuse.Scripting.JavaScript
 
 		void EnsureClassInstanceRooted(Scripting.Context c)
 		{
-			if (_classInstance == null) _classInstance = ((JSContext)c).GetClassInstance(_names);
-			_classInstance.EnsureRooted(c);
+			if (_names != null)
+			{
+				if (_classInstance == null) _classInstance = ((JSContext)c).GetClassInstance(_names);
+				_classInstance.EnsureRooted(c);
+			}
```

Comment:
Are we sure _names == null is OK here? Why would we get a null NameTable passed to the constructor? It seems nonsensical to me...